scripts/configure_dev-deps.sh

```diff
@@ -2,9 +2,24 @@
 
 set -ex
 
+function get_go_version {
+    (
+	cd $(dirname "$0")
+	VERSION=$(cat ../go.mod | grep "$1" 2>/dev/null | awk -F " " '{print $2}')
```

Comment:
I think that if we were to use this for the long run, we need to add `pipefail` as well as `export SHELLOPTS`